Common subdirectories: pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f/manual and pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f-2/manual
diff -u pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f/pthread_win32_attach_detach_np.c pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f-2/pthread_win32_attach_detach_np.c
--- pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f/pthread_win32_attach_detach_np.c	2018-08-08 03:47:40.000000000 -0700
+++ pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f-2/pthread_win32_attach_detach_np.c	2022-08-31 11:12:56.332275500 -0700
@@ -82,14 +82,14 @@
     (void) strncat(QuserExDLLPathBuf,
                    "\\QUSEREX.DLL",
                    sizeof(QuserExDLLPathBuf) - strlen(QuserExDLLPathBuf) - 1);
-    __ptw32_h_quserex = LoadLibrary(QuserExDLLPathBuf);
+    //__ptw32_h_quserex = LoadLibrary(QuserExDLLPathBuf);
   }
 #else
 #  if ! defined(WINCE)
   if(GetSystemDirectory(QuserExDLLPathBuf, sizeof(QuserExDLLPathBuf)/sizeof(TCHAR)) &&
       0 == _tcsncat_s(QuserExDLLPathBuf, _countof(QuserExDLLPathBuf), TEXT("\\QUSEREX.DLL"), 12))
     {
-      __ptw32_h_quserex = LoadLibrary(QuserExDLLPathBuf);
+      //__ptw32_h_quserex = LoadLibrary(QuserExDLLPathBuf);
     }
 #  endif
 #endif
Common subdirectories: pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f/tests and pthreads4w-code-07053a521b0a9deb6db2a649cde1f828f2eb1f4f-2/tests
